consensus/ibft/ibft.go

```diff
@@ -637,12 +646,28 @@ func (i *Ibft) runValidateState() {
 			i.logger.Error("failed to insert block", "err", err)
 			i.handleStateErr(errFailedToInsertBlock)
 		} else {
+			// update metrics
+			i.updateMetrics(block)
+
 			// move ahead to the next block
 			i.setState(AcceptState)
 		}
 	}
 }
+func (i *Ibft) updateMetrics(block *types.Block) {
```

Comment:
Please document this method better

Reply:
Done, commit [link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/40e745fb9520b6e8aaae856a1f0a978803abe2ec)